test/cli_network_test.go

```diff
@@ -24,10 +24,10 @@ func init() {
 // SetUpSuite does common setup in the beginning of each test suite.
 func (suite *PouchNetworkSuite) SetUpSuite(c *check.C) {
 	SkipIfFalse(c, environment.IsLinux)
-	PullImage(c, busyboxImage)
-
-	// Remove all Containers, in case there are legacy containers connecting network.
-	environment.PruneAllContainers(apiClient)
+	//PullImage(c, busyboxImage)
```

Comment:
could you explain why we change this？

Reply:
It is for local tests and I forget to change this code. I have fix this.